.jenkins/actions/run_tests_cmd.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
I would inline this in the `run_tests.sh` to keep things together using...
```bash
cat > /tmp/run_tests_cmd_$$.sh<<EOF
#!/bin/bash
pip3 install --upgrade wheel
 python3 -m pip install --user virtualenv
 python3 -m venv gt4pyenv
 source gt4pyenv/bin/activate
 pip3 install pytest==5.2
 git clone https://github.com/gridtools/gt4py.git gt4py
 pip3 install --no-cache-dir -e gt4py
 python3 -m gt4py.gt_src_manager install
 python3 -m pytest gt4py
EOF

Reply:
Oh good call! with the suggested changes to buildenv I was able to remove the extra layer, at least for this tests. Some tests in some repos that use the schedulerTools may want to do a 2 step thing though if they want to change more of the knobs in the slurm script